electron/execution.ts

```diff
@@ -125,8 +125,7 @@ function onRecordJourneys(mainWindowEmitter: EventEmitter) {
       const handleMainClose = async () => {
         actionListener.removeAllListeners();
         ipc.removeListener('stop', closeBrowser);
-        await browser.close();
-        isBrowserRunning = false;
+        return browser.close();
```

Comment:
can you throw errors in this listener and check if the `isBrowserRunning` variable is rest in the finally block?

Reply:
If I understand this code correctly any errors from this handler should get thrown on their own right? `isBrowserRunning` is reset in the finally block by default already.